### ISSUE TITLE
How can I find the original real images？

### ISSUE THREAD

**Issue by Accepted-Coder** [jtchen0528/PCL-I2G#3]
Excuse me，could you please tell me how I can find the original real images？how can I use the FaceForensics++？And I don't know why I need extract frames from videos (with ffmpeg) at the first step of preprocessing. Thank you!

**Comment by jtchen0528** [comment]
Hi, 

In the original paper, they used FaceForensics++ real videos as their training data. 
In Section 4.1, preprocessing, they said that: For each raw video frame, face crops are
detected and tracked by using [26] and landmarks are detected by public toolbox [4].

FaceForensics++ is a forensics dataset consisting of 1000 original video sequences that have been manipulated with four automated face manipulation methods. They are videos, but the model only takes 2D images as input rather than 3D video sequences. That is why you need a tool to get each frames from the videos. In this case, I use ffmpeg.

You can download FF++ dataset from their github. I've pasted the link in the readme.

Good luck!

**Comment by Accepted-Coder** [comment]
> 

Thank you for your reply! I seem to understand a little. However, I think the FF++ video sequences that have been manipulated are not the real videos. What is the meaning of "real" here?

**Comment by jtchen0528** [comment]
Hi, 

Real means the videos or images are not being deepfaked or manipulated.
I believe FF++ contains real (pristine) videos in the dataset.

https://github.com/ondyari/FaceForensics/tree/master/dataset
under original_sequences folder.

**Comment by Accepted-Coder** [comment]
> Hi,
> 
> Real means the videos or images are not being deepfaked or manipulated. I believe FF++ contains real (pristine) videos in the dataset.
> 
> https://github.com/ondyari/FaceForensics/tree/master/dataset under original_sequences folder.

Thank you! I find the original_sequences folder. I am sorry that I don't know How many 2D images need to train as the input of I2G and what is the size of the images.

**Comment by jtchen0528** [comment]
Hi,

The dataset size is not specified in the paper, however, for FF++, the dataset has train, val, test labels for each pair of videos (real/manipulated). I assume that they used the train set in FF++ as their training data.
Size of images: 256×256 (in the paper, 4.1 section). I use the same size setting.
